image/templates/helm/stackrox-central/templates/01-central-13-deployment.yaml

```diff
@@ -48,18 +48,18 @@ spec:
                     values:
                     - "true"
             {{- if ._rox.env.openshift }}
-            - weight: 25
+            - weight: 50
```

Comment:
Just curious why not keep it at 75?

Reply:
It could be 75. Basically it could be any number higher than 25 (`compute`'s weight)